pulp_container/app/migrations/0033_raise_warning_for_repair.py

```diff
@@ -0,0 +1,21 @@
+import warnings
+
+from django.db import migrations
+
+
+def print_warning_for_repair(apps, schema_editor):
```

Comment:
before raising warning you should first query the DB and check for the presence of the schema v1 manifests.